po/nl.po

```diff
@@ -0,0 +1,550 @@
+msgid "Smooth Scrolling"
+msgstr "Vloeiend scrollen"
+
+#: lib/logitech_receiver/settings_templates.py:120
+#: lib/logitech_receiver/settings_templates.py:125
+#: lib/logitech_receiver/settings_templates.py:129
+msgid "High-sensitivity mode for vertical scroll with the wheel."
+msgstr "Hogegevoeligheidsmodus voor verticaal scrollen."
+
+#: lib/logitech_receiver/settings_templates.py:121
+msgid "HID++ Scrolling"
+msgstr "HID++-scrollen"
+
+#: lib/logitech_receiver/settings_templates.py:122
+msgid "HID++ mode for vertical scroll with the wheel."
+msgstr "HID++-modus voor verticaal scrollen."
+
+#: lib/logitech_receiver/settings_templates.py:124
+msgid "High Resolution Scrolling"
+msgstr "Scrollen met hoge resolutie"
+
+#: lib/logitech_receiver/settings_templates.py:126
+msgid "High Resolution Wheel Invert"
+msgstr "Andersom scrollen met hoge resolutie"
+
+#: lib/logitech_receiver/settings_templates.py:127
+msgid "High-sensitivity wheel invert mode for vertical scroll."
+msgstr "Hogegevoeligheidsmodus voor andersom verticaal scrollen."
+
+#: lib/logitech_receiver/settings_templates.py:128
+msgid "Wheel Resolution"
+msgstr "Scrollresolutie"
+
+#: lib/logitech_receiver/settings_templates.py:130
+msgid "Side Scrolling"
+msgstr "Zijwaarts scrollen"
+
+#: lib/logitech_receiver/settings_templates.py:131
+msgid ""
+"When disabled, pushing the wheel sideways sends custom button events\n"
+"instead of the standard side-scrolling events."
+msgstr ""
+"Schakel dit uit om aangepaste knopgebeurtenissen te versturen als u het "
+"scrollwiel\n"
+"opzij duwt in plaats van de standaard gebeurtenissen."
+
+#: lib/logitech_receiver/settings_templates.py:133
+msgid "Sensitivity (DPI)"
+msgstr "Gevoeligheid (DPI)"
+
+#: lib/logitech_receiver/settings_templates.py:134
+msgid "Sensitivity (Pointer Speed)"
+msgstr "Gevoeligheid (cursorsnelheid)"
+
+#: lib/logitech_receiver/settings_templates.py:135
+msgid "Swap Fx function"
+msgstr "Fx-functies omdraaien"
+
+#: lib/logitech_receiver/settings_templates.py:136
+msgid ""
+"When set, the F1..F12 keys will activate their special function,\n"
+"and you must hold the FN key to activate their standard function."
+msgstr ""
+"Schakel dit in om de speciale functies van F1-F12 te gebruiken -\n"
+"de standaard functies kunnen worden geactiveerd met Fn + F1-F12."
+
+#: lib/logitech_receiver/settings_templates.py:139
+msgid ""
+"When unset, the F1..F12 keys will activate their standard function,\n"
+"and you must hold the FN key to activate their special function."
+msgstr ""
+"Schakel dit in om de standaard functies van F1-F12 te gebruiken -\n"
+"de speciale functies kunnen worden geactiveerd met Fn + F1-F12."
+
+#: lib/logitech_receiver/settings_templates.py:141
+msgid "Hand Detection"
+msgstr "Handpalmdetectie"
+
+#: lib/logitech_receiver/settings_templates.py:142
+msgid "Turn on illumination when the hands hover over the keyboard."
+msgstr ""
+"Schakelt de verlichting in als u beide handen boven het toetsenbord houdt."
```

Comment:
Één hand is ook al voldoende hoor :)

Reply:
Ah, dat wist ik niet! Ik heb zelf geen toetsenbord uit die serie, dus ik ging uit van "hands", zoals het er in de oorspronkelijke tekst stond. Bedankt voor de correctie; ook dit ga ik direct verbeteren!